jib-gradle-plugin/README.md

```diff
@@ -128,6 +128,8 @@ eval $(minikube docker-env)
 gradle jibDockerBuild
 ```
 
+See [`dockerClient`](#dockerclient-closure) for more configuration options.
```

Comment:
The doc was talking about doing `eval $(minikube docker-env)`, for which you have an alternative to set `dockerClient.environment`. What I mean is, it may be confusing to users if there are two different ways to achieve the same goal, so I think it's good to clear any potential confusion.

What if we add "Alternatively, you can set environment variables in the Jib configuration. See ... "